File: 58. Length of Last Word/lengthoflastword.java
class Solution {
    public int lengthOfLastWord(String s) {
        int length = s.length();
        int i = length - 1;

        while (i >= 0 && s.charAt(i) == ' ') {
            i--;
        }

        int count = 0;

        while (i >= 0 && s.charAt(i) != ' ') {
            count++;
            i--;
        }
        
        return count;
    }
}

File: 58. Length of Last Word/lengthoflastword.c
int lengthOfLastWord(char* s) {
    int length = strlen(s);
    int i = length - 1;

    while (i >= 0 && s[i] == ' ') {
        i--;
    }

    int count = 0;

    while (i >= 0 && s[i] != ' ') {
        count++;
        i--;
    }
    
    return count;
}
